client/state/components/drop-zone/test/selectors.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { isDropZoneVisible } from '../selectors';
+
+describe( 'selectors', () => {
+	it( 'should return false as a default state', () => {
+		expect( isDropZoneVisible( { dropZone: undefined } ) ).to.be.eql( false );
```

Comment:
I'm a bit 50/50 on that :) On one hand it's more readable syntax, on the other hand it's an IDE linter warning, because it's not a call or assignment. 

Going for the more readable syntax for now.